baselines/Google Meet Minimum Viable Secure Configuration Baseline v0.2.md

```diff
@@ -190,3 +190,41 @@ To enable Host Management meeting features:
 3.  Select **Meet safety settings** -\> **Warn for external participants**.
 4.  Check the **External or unidentified participants in a meeting are given a label** checkbox.
 5.  Select **Save**.
+
+## 5. 1:1 Calling
+
+This section covers who domain users are allowed to receive a 1:1 call from.
+
+### Policies
+
+#### GWS.CHAT.5.1v0.1
+Users receive calls only from contacts and other users in the organization SHALL be selected.
+
+- _Rationale:_ Calls could potentially be used to pass sensitive information. By selecting this setting, it potentially mitigates unauthorized data leakage.
+- Last Modified: March 15, 2024
+
+- MITRE ATT&CK TTP Mapping
+  - [T1530: Data from Cloud Storage](https://attack.mitre.org/techniques/T1530/)
```

Comment:
I'm not sure if "Data from Cloud Storage" applies here.

Reply:
@adhilto Here is the response from the MITRE member who does the TTP mappings for the baselines:

Just looked through this policy. I didn't do the original mapping for this but i do think that Data from Cloud Storage is applicable here and SHOULD NOT be removed. Reasoning: Data from Cloud storage is one of the broadest applicable TTP mapping in ATT&CK. Any time there is a threat of data being leaked or removed (intentionally/unintentionally) from cloud storage through ANY means, Data from cloud storage applies. In this case, data could be leaked from cloud storage if 1:1 calls are allowed from non-contacts. An employee could very well send sensitive information from within GWS to the unauthorized caller. Also, our SCB rationale supports this:
"Calls could potentially be used to pass sensitive information. By selecting this setting, it potentially mitigates unauthorized data leakage". If you guys still think that it should be removed, we can. There isn't any harm in removing it since other mappings are providing coverage. Let me know. 
 
Direct quote from ATT&CK page for this:
Adversaries may collect sensitive data from these cloud storage solutions. Providers typically offer security guides to help end users configure systems, though misconfigurations are a common problem.[[1]](https://aws.amazon.com/premiumsupport/knowledge-center/secure-s3-resources/)[[2]](https://docs.microsoft.com/en-us/azure/storage/common/storage-security-guide)[[3]](https://cloud.google.com/storage/docs/best-practices) There have been numerous incidents where cloud storage has been improperly secured, typically by unintentionally allowing public access to unauthenticated users, overly-broad access by all users, or even access for any anonymous person outside the control of the Identity Access Management system without even needing basic user permissions.
This open access may expose various types of sensitive data, such as credit cards, personally identifiable information, or medical records.[[4]](https://www.trendmicro.com/vinfo/us/security/news/virtualization-and-cloud/a-misconfigured-amazon-s3-exposed-almost-50-thousand-pii-in-australia)[[5]](https://www.wired.com/story/magecart-amazon-cloud-hacks/)[[6]](https://www.hipaajournal.com/47gb-medical-records-unsecured-amazon-s3-bucket/)[[7]](https://redcanary.com/blog/rclone-mega-extortion/)
Adversaries may also obtain then abuse leaked credentials from source repositories, logs, or other means as a way to gain access to cloud storage objects.

Reply:
I don't feel very strongly on this point, I'll defer to their judgement.

Reply:
@jkaufman-mitre @adhilto 
After further discussion, maybe we could add this instead of cloud storage, I think it's more applicable https://attack.mitre.org/techniques/T1598/004/

Reply:
Please ignore the above, I see it's already there lol